core/CMakeLists.txt

```diff
@@ -96,6 +96,7 @@ detray_add_library( detray_core core
    "include/detray/surface_finders/grid/detail/serializer_impl.hpp"
```

Comment:
I think you are missing `container_views.hpp` and `viewable_collection.hpp` here

Reply:
Absolutely correct, I will fix it while I rebase